src/Coordination/Changelog.h

```diff
@@ -144,7 +152,9 @@ class Changelog
     /// Mapping log_id -> log_entry
     IndexToLogEntry logs;
     /// Start log_id which exists in all "active" logs
-    uint64_t start_index = 0;
+    /// min_log_id + 1 == max_log_id means empty log storage for NuRaft
+    uint64_t min_log_id = 0;
```

Comment:
As I see we only assign this variable and compare with `0`, shoul it really be `int`? Maybe just flag like `read_some` ?

Reply:
It's required in `uint64_t getStartIndex()`